apps/picsa-apps/dashboard/src/app/modules/resources/pages/home/resources.page.html

```diff
@@ -1,4 +1,4 @@
-<div class="page-content">
+                        <div class="page-content">
```

Comment:
nit(blocking)
I'm not sure why this whitespace has been added. It would be good if you could set up a code formatter to automatically handle issues like these. I would recommend installing the _prettier_ extension for vscode and enabling _format on save_ within vscode settings.

Reply:
@chrismclarke thank you for the pointer, i will look into it.